src/renderer/components/ft-search-filters/ft-search-filters.vue

```diff
@@ -48,6 +48,16 @@
           class="searchRadio"
           @change="updateDuration"
         />
+        <ft-checkbox-list
+          ref="featuresCheck"
+          :title="$t('Search Filters.Features.Features')"
+          :labels="featureLabels"
+          :values="featureValues"
+          :initial-values="searchDefaultFeatures"
+          class="searchRadio"
+          :is-checkbox="true"
```

Comment:
This prop doesn't exist on the `ft-checkbox-list` component, so it can be removed.
```suggestion
```